telegram/bot.py

```diff
@@ -2042,22 +2049,22 @@ def answer_inline_query(
         @no_type_check
         def _set_defaults(res):
             # pylint: disable=W0212
-            if res._has_parse_mode and res.parse_mode == DEFAULT_NONE:
+            if hasattr(res, 'parse_mode') and res.parse_mode == DEFAULT_NONE:
                 if self.defaults:
                     res.parse_mode = self.defaults.parse_mode
                 else:
                     res.parse_mode = None
-            if res._has_input_message_content and res.input_message_content:
+            if hasattr(res, 'input_message_content') and res.input_message_content:
                 if (
-                    res.input_message_content._has_parse_mode
+                    hasattr(res.input_message_content, 'parse_mode')
                     and res.input_message_content.parse_mode == DEFAULT_NONE
                 ):
                     if self.defaults:
                         res.input_message_content.parse_mode = self.defaults.parse_mode
```

Comment:
IISC it's not really necessary as `Defaults.parse_mode` defaults to `None` (instead of `DEFAULT_NONE`), but it sure doesn't hurt to have it there. added it everywhere, where applicable.